app/src/main/java/com/kickstarter/viewmodels/CommentsViewModel.kt

```diff
@@ -301,7 +305,7 @@ interface CommentsViewModel {
                     .distinctUntilChanged(true)
                     .startOverWith(startOverWith)
                     .envelopeToListOfData {
-                        mapToCommentCardDataList(Pair(it, null))
+                        mapToCommentCardDataList(Pair(it, this.project))
```

Comment:
Since project can technically be null, I think it would be best if we put a null check here to guarantee that no crash happens.

Reply:
Changed map method to not allow null